tldw_Server_API/tests/Evaluations/test_recipe_runs_jobs_worker.py

```
@@ -0,0 +1,539 @@
+                        recipe_id=record.recipe_id,
+                        owner_user_id=user_id,
+                    ),
+                },
+                db=db,
+                user_id=user_id,
+                service=_BrokenService(),
+            )
+        finally:
+            monkeypatch.undo()
+    except RuntimeError as exc:
+        assert str(exc) == "boom"
```

Ruff (0.15.7)

[warning] Found assertion on exception `exc` in `except` block, use `pytest.raises()` instead

(PT017)